components/Book.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import {
+  Image,
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity
+} from 'react-native';
+import { Button} from './common';
+import firebase from 'firebase';
+import { loginDispatch, loginDispatchFalse } from '../redux/actions/authActions'
+import { connect } from 'react-redux';
+
+
+class Book extends Component {
+    componentWillMount(){
+      firebase.auth().onAuthStateChanged((user) => {
+          console.log((this.props, ' in authfirebase', user))
+          if (user) {
+            this.props.loginDispatch(user.uid)
+          }
+          else this.props.loginDispatchFalse()
+        })
+      }
+  
+handleAddBooks=(book)=>{
```

Comment:
handle book needs to call a function from redux file ---> such as this.props.handleBookFromREdux()

Reply:
agreed